From 1f6c1a9bda7d63626249c481a22ff286e7da7dd3 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Fri, 9 Mar 2012 22:19:19 +0530
Subject: [PATCH 138/186] ARM: OMAP2+: nand: Fix build breakage.

This patch fixes the build breakage

arch/arm/mach-omap2/board-flash.c: In function 'board_flash_init':
arch/arm/mach-omap2/board-flash.c:246:16: error:
'nand_default_timings' undeclared (first use in this function)
arch/arm/mach-omap2/board-flash.c:246:16: note: each undeclared
identifier is reported only once for each function it appears in
make[1]: *** [arch/arm/mach-omap2/board-flash.o] Error 1
make: *** [arch/arm/mach-omap2] Error 2
make: *** Waiting for unfinished jobs....

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/board-flash.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-omap2/board-flash.c b/arch/arm/mach-omap2/board-flash.c
index caa1e82..915ec79 100644
--- a/arch/arm/mach-omap2/board-flash.c
+++ b/arch/arm/mach-omap2/board-flash.c
@@ -104,9 +104,6 @@ __init board_onenand_init(struct mtd_partition *nor_parts, u8 nr_parts, u8 cs)
 }
 #endif /* CONFIG_MTD_ONENAND_OMAP2 || CONFIG_MTD_ONENAND_OMAP2_MODULE */
 
-#if defined(CONFIG_MTD_NAND_OMAP2) || \
-		defined(CONFIG_MTD_NAND_OMAP2_MODULE)
-
 /* Note that all values in this struct are in nanoseconds */
 struct gpmc_timings nand_default_timings = {
 
@@ -131,6 +128,9 @@ struct gpmc_timings nand_default_timings = {
 	.wr_data_mux_bus = 0,
 };
 
+#if defined(CONFIG_MTD_NAND_OMAP2) || \
+		defined(CONFIG_MTD_NAND_OMAP2_MODULE)
+
 static struct omap_nand_platform_data omap_nand_data = {
 	.gpmc_t		= &nand_default_timings,
 };
-- 
1.7.9.1

